unable to run application on develop branch

**Describe the bug**
Unable to run application on develop branch
 
**To Reproduce**
Steps to reproduce the behavior:
1.  checkout to develop branch  
2. Build and run


**Expected behavior**
Application runs 



**Smartphone (please complete the following information):**
   Tested on multiple devices
 - Device: Pixel 3a
 - OS: Android 11.0
 - Android API 30

**Additional context**
    --------- beginning of crash
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: openfoodfacts.github.scrachx.openfood.debug, PID: 5160
    java.lang.RuntimeException: Unable to get provider androidx.startup.InitializationProvider: java.lang.ClassNotFoundException: Didn't find class "androidx.startup.InitializationProvider" on path: DexPathList[[zip file "/data/app/~~5yObK6mivKBYqqZ_vUVd_Q==/openfoodfacts.github.scrachx.openfood.debug-2FZVTRen73Jo50niTI_HOg==/base.apk"],nativeLibraryDirectories=[/data/app/~~5yObK6mivKBYqqZ_vUVd_Q==/openfoodfacts.github.scrachx.openfood.debug-2FZVTRen73Jo50niTI_HOg==/lib/x86, /data/app/~~5yObK6mivKBYqqZ_vUVd_Q==/openfoodfacts.github.scrachx.openfood.debug-2FZVTRen73Jo50niTI_HOg==/base.apk!/lib/x86, /system/lib, /system_ext/lib]]
        at android.app.ActivityThread.installProvider(ActivityThread.java:7244)
        at android.app.ActivityThread.installContentProviders(ActivityThread.java:6780)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6697)
        at android.app.ActivityThread.access$1300(ActivityThread.java:237)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1913)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
     Caused by: java.lang.ClassNotFoundException: Didn't find class "androidx.startup.InitializationProvider" on path: DexPathList[[zip file "/data/app/~~5yObK6mivKBYqqZ_vUVd_Q==/openfoodfacts.github.scrachx.openfood.debug-2FZVTRen73Jo50niTI_HOg==/base.apk"],nativeLibraryDirectories=[/data/app/~~5yObK6mivKBYqqZ_vUVd_Q==/openfoodfacts.github.scrachx.openfood.debug-2FZVTRen73Jo50niTI_HOg==/lib/x86, /data/app/~~5yObK6mivKBYqqZ_vUVd_Q==/openfoodfacts.github.scrachx.openfood.debug-2FZVTRen73Jo50niTI_HOg==/base.apk!/lib/x86, /system/lib, /system_ext/lib]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:207)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at android.app.AppComponentFactory.instantiateProvider(AppComponentFactory.java:147)
        at androidx.core.app.CoreComponentFactory.instantiateProvider(CoreComponentFactory.java:67)
        at android.app.ActivityThread.installProvider(ActivityThread.java:7228)
        at android.app.ActivityThread.installContentProviders(ActivityThread.java:6780) ?
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6697) ?
        at android.app.ActivityThread.access$1300(ActivityThread.java:237) ?
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1913) ?
        at android.os.Handler.dispatchMessage(Handler.java:106) ?
        at android.os.Looper.loop(Looper.java:223) ?
        at android.app.ActivityThread.main(ActivityThread.java:7656) ?
        at java.lang.reflect.Method.invoke(Native Method) ?
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592) ?
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947) ?

--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -64,6 +64,7 @@ dependencies {
     implementation("androidx.legacy:legacy-support-v4:1.0.0")
     implementation("androidx.work:work-runtime:2.4.0")
     implementation("androidx.work:work-rxjava2:2.4.0")
+    implementation ("androidx.startup:startup-runtime:1.0.0")
 
 
     kapt("com.google.dagger:dagger-compiler:2.30.1")

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -57,6 +57,11 @@
             android:name=".utils.SearchSuggestionProvider"
             android:authorities="${applicationId}.utils.SearchSuggestionProvider" />
 
+        <provider
+            android:name="androidx.startup.InitializationProvider"
+            android:authorities="${applicationId}.androidx-startup"
+            tools:node="remove" />
+
         <activity
             android:name=".features.splash.SplashActivity"
             android:label="@string/app_name"

